csharp/src/Microsoft.ML.OnnxRuntime/ThreadingOptions.Shared.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.ML.OnnxRuntime
+{
+    public class ThreadingOptions : SafeHandle
```

Comment:
> public class ThreadingOptions : SafeHandle [](http://example.com/codeflow?start=3&length=43)

This will fail documentation check. Please, refer to other classes. At a very least every class and every public member must be documented.

Reply:
To run the documentation check locally you would need to install doxygen and run it similar to what is in our pipelines:

Point ORT_DOXY_SRC to the location of the source code.
Point ORT_DOXY_OUT the location of the doxygen output folder.

set ORT_DOXY_SRC=$(Build.SourcesDirectory)
set ORT_DOXY_OUT=$(Build.BinariesDirectory)\${{ parameters.BuildConfig }}\${{ parameters.BuildConfig }}
mkdir %ORT_DOXY_SRC%
 mkdir %ORT_DOXY_OUT%
     "C:\Program Files\doxygen\bin\doxygen.exe" $(Build.SourcesDirectory)\tools\ci_build\github\Doxyfile_csharp.cfg